app/schemas/models/user.coffee

```diff
@@ -278,6 +278,7 @@ _.extend UserSchema.properties,
   schoolName: {type: 'string', description: 'Deprecated string. Use "school" object instead.'}
   role: {type: 'string', enum: ["advisor", "parent", "principal", "student", "superintendent", "teacher", "technology coordinator", "possible teacher"]}  # unset: home player
   verifiedTeacher: { type: 'boolean' }
+  schoolAdmin: { type: 'boolean' }
```

Comment:
Let's add a description here to describe what this means.

Reply:
Do we also need a `administratedTeachers` field in user schema?

Reply:
Do we need a separate `schoolAdmin` field if it can be deduced from an empty or undefined `administratedTeachers` field?

Reply:
Oops, left over from a previous implementation where I had this field to work on the toggle. We get this from the permissions 👍 

And yes, let me add it for the `administratedTeachers` field.